browser/help/topics/v3-ht-annotations.md

```diff
@@ -31,8 +35,24 @@ The gnomAD v3 Hail Table annotations are defined below:
   - **bin_freq**: Bin frequencies for the age histogram. This is the number of records found in each bin.
   - **n_smaller**: Count of age values falling below lowest histogram bin edge.
   - **n_larger**: Count of age values falling above highest histogram bin edge.
-- **freq_sample_count**: A sample count per sample grouping defined in the 'freq_meta' global annotation.
-
+- **freq_meta_sample_count**: A sample count per sample grouping defined in the 'freq_meta' global annotation.
+- **vrs_version**: The Variant Representation Specification version that was used to compute IDs on the callset.
```

Comment:
minor note that plural (vrs_versions) is more consistent with `tool_versions` below (particularly given this has not just the VRS version but also the seqrepo version now)